recipe_scrapers/__version__.py

```diff
@@ -1 +1 @@
-__version__ = "14.35.0"
```

Comment:
Hi @maraid - as a notification: we often bundle multiple pull requests together for releases (not always; it's somewhat unpredictable), and so each pull request generally shouldn't update the version number individually.

Because that's a small easy fix I'll likely undo that change here while reviewing the pull request.